configs/config.yml

```diff
@@ -0,0 +1,34 @@
+log:
+  level: debug
+  pretty: true
+
+poller:
+  enabled: true
+  interval: 3000
+  blocks_per_poll: 3
+  from_block: 20000000
+  until_block: 20000010
+
+commiter:
+  enabled: true
+  interval: 10000
+  batch_size: 100
+
+failure_recoverer:
+  enabled: true
+  interval: 3000
+  batch_size: 100
+
+storage:
+  main:
+    driver: "clickhouse"
+    clickhouse:
+      port: 9440
+      database: "base"
+  staging:
+    driver: "clickhouse"
+    clickhouse:
+      port: 9440
+      database: "staging"
+  orchestrator:
+    driver: "memory"
```

Comment:
don't we need the maxItem config for memory?

Reply:
the code sets defaults to most values. Default here is 1000